build/format-notecards.js

```diff
@@ -0,0 +1,15 @@
+// Remove h4s from any existing notecards and transform them
+// from <div class="note notecard"><h4>Note:</h4>foobar</div> to
+// <div class="note notecard"><p><strong>Note:</strong>foobar</p></div>
+function formatNotecards($) {
+  $(".note.notecard h4").each((_, element) => {
+    const h4 = $(element);
+    const text = h4.text();
+    const p = $("p:first", h4.parents("div.notecard"));
+    p.html(`<strong>${text}:</strong> ${p.html()}`);
+    h4.empty();
+    h4.remove();
```

Comment:
Oh. I guess the `h4.remove()` on the line above is useless because it's going to be removed on the next line.

Reply:
Agreed. Just a single `h4.remove()` is needed.